.github/workflows/integration_test.yaml

```diff
@@ -43,7 +59,8 @@ jobs:
           docker tag $imageId localhost:32000/gunicorn:latest
           docker push localhost:32000/gunicorn:latest
       - name: Run integration tests
-        run: tox -e integration -- --model testing --gunicorn-image localhost:32000/gunicorn:latest --statsd-prometheus-exporter-image prom/statsd-exporter
+        run: |
+          tox -e integration -- --gunicorn-image localhost:32000/gunicorn:latest --statsd-prometheus-exporter-image prom/statsd-exporter --influx-model-name influxmodel
```

Comment:
You don't need to pass the prometheus exporter as an argument. Check https://github.com/canonical/indico-operator/blob/main/tests/integration/conftest.py#L27. Make sure it is defined in the metadata file as in https://github.com/canonical/indico-operator/blob/main/metadata.yaml#L34

Reply:
You can probably do something similar for influx-model-name arg

Reply:
Nice comment @arturo-seijas , I had to dig into the Indico source to figure that out :rofl: